index.html

```diff
@@ -1,16 +1,16 @@
 ---
 layout: layout.liquid
-title: Open source key-value database on blockchain
+title: Open Source on Blockchain Immutable Meta/Data
```

Comment:
I like this change, "kye-value" is a technical term and most people don't know what it is.

How about:
Open Source Blockchain Metadata

"immutable" is a double-edge sword, some like it, some may not.